packages/storybook/src/stories/changelog/button.stories.mdx

```diff
@@ -0,0 +1,9 @@
+import { Meta } from '@storybook/addon-docs'
```

Comment:
@eeeeean can you check if these files are in your `.gitignore`?

Reply:
Gotcha, then I recommend doing a rebase and verifying that the `changelog/*.stories.mdx` files are not checked in: https://github.com/little-spoon-dev/design-system/blob/2bff3e2/packages/storybook/.gitignore

Reply:
Confirming I see `src/stories/changelog/` in `storybook/.gitignore`

Reply:
Nice, can you rebase and remove all the files under `src/stories/changelog/`?